packages/cascara/src/ui/Table/TableBase.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
```

Comment:
This is all of the display logic previously in Table so we only run it when we are not trying to render Loading or Empty states